### ISSUE TITLE
Duplicate Host header requirements (Trac #91)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#91]
Section 9:

```text
The set of common methods for HTTP/1.1 is defined below. Although
this set can be expanded, additional methods cannot be assumed to
share the same semantics for separately extended clients and servers.
                          
The Host request-header field (section 14.23) MUST accompany all
HTTP/1.1 requests.
```
            
...any reason why the Host header requirement is listed here so prominently (duplicating text from 14.23)?

Migrated from https://trac.ietf.org/ticket/91
```json
{
    "status": "closed",
    "changetime": "2012-03-22T00:03:06",
    "_ts": "1332374586898580",
    "description": "Section 9:\n\n{{{\nThe set of common methods for HTTP/1.1 is defined below. Although\nthis set can be expanded, additional methods cannot be assumed to\nshare the same semantics for separately extended clients and servers.\n                          \nThe Host request-header field (section 14.23) MUST accompany all\nHTTP/1.1 requests.\n}}}\n            \n...any reason why the Host header requirement is listed here so prominently (duplicating text from 14.23)?\n\n",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T03:51:27",
    "component": "non-specific",
    "summary": "Duplicate Host header requirements",
    "priority": "normal",
    "keywords": "",
    "milestone": "02",
    "owner": "fielding@gbiv.com",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20071222T03:51:42: mnot@pobox.com changed description from:

> Section 9:</p>
> 
> {{{
> The set of common methods for HTTP/1.1 is defined below. Although
> this set can be expanded, additional methods cannot be assumed to
> share the same semantics for separately extended clients and servers.
>                           
> The Host request-header field (section 14.23) MUST accompany all
> HTTP/1.1 requests.
> }}}
>             
> ...any reason why the Host header requirement is listed here so prominently (duplicating text from 14.23)?
> 
> 

to:

> Section 9:
> 
> {{{
> The set of common methods for HTTP/1.1 is defined below. Although
> this set can be expanded, additional methods cannot be assumed to
> share the same semantics for separately extended clients and servers.
>                           
> The Host request-header field (section 14.23) MUST accompany all
> HTTP/1.1 requests.
> }}}
>             
> ...any reason why the Host header requirement is listed here so prominently (duplicating text from 14.23)?
> 
>

**Comment by mnot** [comment]
Trac update at 20080104T06:01:41: 
* mnot@pobox.com changed component from "" to "auth"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080104T06:25:02: 
* mnot@pobox.com changed component from "auth" to "non-specific"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080105T01:50:01: 
* fielding@gbiv.com commented:

> The requirement here is redundant.


* fielding@gbiv.com changed milestone from "unassigned" to "02"
* fielding@gbiv.com changed owner from "" to "fielding@gbiv.com"
* fielding@gbiv.com changed status from "new" to "assigned"

**Comment by mnot** [comment]
Trac update at 20080205T00:18:19: 
* fielding@gbiv.com commented:

> Fixed by removing the paragraph in [195].
> 


* fielding@gbiv.com changed resolution from "" to "fixed"
* fielding@gbiv.com changed status from "assigned" to "closed"

**Comment by mnot** [comment]
Trac update at 20080205T09:13:41: julian.reschke@gmx.de commented:

> From [201]:
> 
> Record fix in Changes section, related to #91.

**Comment by mnot** [comment]
Trac update at 20120322T00:03:06: 
* mnot@pobox.com changed priority from "" to "normal"
* mnot@pobox.com changed severity from "" to "Active WG Document"
